### PR TITLE
Update mural modals and headshots

### PR DESCRIPTION
**Pull request by johncip**
This PR mostly makes improvements to the mural modals and thumbnails.

localhost_5000_artwork html (2)

Artwork page content:
- **adds more murals** (see [here](https://docs.google.com/spreadsheets/d/1D81nVcmuWDlhQ1C3DR_ZSW3WVZ9kNelT0u5DsQAkQUo/edit#gid=0) for progress -- all the ones I can find photos for are done)
- **adds hand-cropped mural thumbnails**
- updates lead artists, addresses, titles where incorrect
- fixes format of date (remove comma)
- removes an FAQ from the artwork page that said "coming soon"

Artwork page code & styling:
- improves centering of inline icons
- fixes an issue where the white "caption" area was still sometimes not wide enough to contain the close button
- simplifies internal (data-) names and image filenames
- removes unused `<p>` tags

About us page:
- restores Erin Bakke to the about us page (also removed background on pic)
- update Ashley and Andre's pictures to new pics sent by Ashley
- crop Keith's picture better

**This also removes the use of `.muralGallery` (hides mural thumbnails after the first 6). Can't decide if we should just leave this off, at least on desktop. But I want it off temporarily at least.**

### CODE REVIEW

File: artwork.html [hunk 1]
```diff
@@ -275,200 +359,437 @@ <h2 class="sectionHeading sectionHeading-centered sectionHeading-spaced">Frequen
...
+            </div>
+            <div class="muralModalContainer" data-mural-name="sph">
+                <div class="muralModal">
+                    <div class="muralModal--img"><img src="images/artwork/sph.jpg"></div>
+                    <div class="muralModal--caption">
+                        <h2 class="muralModal--name">San Pablo Hotel</h2>
+                        <button class="muralModal--close" aria-label="Close">×</button>
+                        <p>
+                            <img class="muralModal--icon" src="images/palette.svg">
+                            <span class="muralModal--bold">Lead Artist</span><span class="muralModal--text">: Derrick "Rtystk" Shavers</span>
```
Comment (coramonokandilos) [inline artwork.html:742]: on the spreadsheet (https://docs.google.com/spreadsheets/d/1RXPV8kqsDCZ6gDK1Jp8YM9o-01NhK7qokUNO9favf80/edit#gid=0) it says that Andre is the artist, not Derrick.

**Review by coramonokandilos — COMMENTED** [review]
I compared the spreadsheet and what exists now in this branch and this is what I found that was missing:

- [ ] Side 2: Learning Occurs (no pictures)
- [ ] Immigration/Migration 
- [ ] Senior Superheros (no pictures)
- [ ] Don't Shoot

You mentioned you did 9/16 - I only caught 4, so not sure where the other 3 are from. Besides the comment I made about the Lead Artist on the San Pablo Hotel mural, all seems good!

**Comment by coramonokandilos** [comment]
Just saw the "Copy of BAMP Murals" you made, it seems like the 4 I pointed out are the same 4 that you have at the bottom of the spreadsheet, so that makes sense.

**Comment by johncip** [comment]
Whoops, I did say 9 but after I sent that I ended up doing some more.

Feel free to swap out the other photos from the email if you have time... I
can't do more today but I will try over the weekend.

Also it looks like the spreadsheet also has Rtystk as the artist for the
San Pablo Hotel mural?

**Comment by johncip** [comment]
Also, in case it wasn't clear from the sheet, I couldn't do the other 4 because I can't find photos of them. But I put on Asana.

**Pull request merged**